src/Component/Shop.js

```diff
@@ -0,0 +1,36 @@
+// eslint-disable-next-line import/named
+import React, { useState, useEffect } from 'react';
+import '../Stylesheet/Shop.css';
+import Products from './Products';
+import { products } from '../DataModel/ProductData';
+import Cart from './Cart';
+import { subscribe, unsubscribe } from './Event';
+
+function Shop() {
+  const [cartItems, setCart] = useState([]);
+  useEffect(() => {
+    subscribe('addToCart', (data) => {
```

Comment:
@prabhatVinsol Use different use effect for adding and removing

Reply:
Done